plutus-use-cases/plutus-use-cases.cabal

```diff
@@ -1,242 +1,262 @@
+    , tasty-hunit
+    , tasty-quickcheck
+    , text
+
+  if !(impl(ghcjs) || os(ghcjs))
+    build-depends: plutus-tx-plugin
 
 -- runs emulator traces from plutus-use-cases-tests and
 -- writes all applied validator scripts to a folder
 executable plutus-use-cases-scripts
-    main-is: Main.hs
-    hs-source-dirs:
-        scripts
-        test
-    default-language: Haskell2010
-    default-extensions: ImportQualifiedPost
-    other-modules:
-        Spec.Auction
-        Spec.Crowdfunding
-        Spec.Currency
-        Spec.ErrorHandling
-        Spec.Escrow
-        Spec.Escrow.Endpoints
-        Spec.SimpleEscrow
-        Spec.Future
-        Spec.Game
-        Spec.GameStateMachine
-        Spec.Governance
-        Spec.MultiSig
-        Spec.MultiSigStateMachine
-        Spec.PingPong
-        Spec.PubKey
-        Spec.Prism
-        Spec.Rollup
-        Spec.Stablecoin
-        Spec.TokenAccount
-        Spec.Vesting
-    ghc-options: -Wall -Wnoncanonical-monad-instances
-                 -Wincomplete-uni-patterns -Wincomplete-record-updates
-                 -Wredundant-constraints -Widentities -rtsopts
-                 -- See Plutus Tx readme
-                 -fobject-code -fno-ignore-interface-pragmas -fno-omit-interface-pragmas
-    build-depends:
-        base >= 4.9 && <5,
-        bytestring -any,
-        containers -any,
-        data-default -any,
-        freer-extras -any,
-        prettyprinter -any,
-        QuickCheck -any,
-        tasty -any,
-        tasty-hunit -any,
-        tasty-golden -any,
-        tasty-quickcheck -any,
-        text -any,
-        lens -any,
-        mtl -any,
-        freer-simple -any,
-        foldl -any,
-        streaming -any,
-        plutus-ledger-api -any,
-        plutus-tx -any,
-        plutus-contract -any,
-        plutus-ledger -any,
-        plutus-ledger-constraints -any,
-        plutus-use-cases -any,
-        plutus-contract-certification -any,
-        plutus-script-utils -any,
-        optparse-applicative -any,
-        cardano-api -any
-
-    if !(impl(ghcjs) || os(ghcjs))
-        build-depends: plutus-tx-plugin -any
+  main-is:            Main.hs
+  hs-source-dirs:     scripts test
+  default-language:   Haskell2010
+  default-extensions: ImportQualifiedPost
+  other-modules:
+    Spec.Auction
+    Spec.Crowdfunding
+    Spec.Currency
+    Spec.ErrorHandling
+    Spec.Escrow
+    Spec.Escrow.Endpoints
+    Spec.Future
+    Spec.Game
+    Spec.GameStateMachine
+    Spec.Governance
+    Spec.MultiSig
+    Spec.MultiSigStateMachine
+    Spec.PingPong
+    Spec.Prism
+    Spec.PubKey
+    Spec.Rollup
+    Spec.SimpleEscrow
+    Spec.Stablecoin
+    Spec.TokenAccount
+    Spec.Vesting
+
+  ghc-options:
+    -Wall -Wnoncanonical-monad-instances -Wincomplete-uni-patterns
+    -Wincomplete-record-updates -Wredundant-constraints -Widentities
+    -rtsopts -fobject-code -fno-ignore-interface-pragmas
+    -fno-omit-interface-pragmas
+
+  -- See Plutus Tx readme
```

Comment:
Hmm, this is unfortunate. The comment applies to the last 3 `ghc-options`, but now it looks like a comment on `build-depends`

Reply:
This is the best I could come up with https://github.com/input-output-hk/plutus-apps/pull/505/commits/1dc70e996147fc4b93c775ce35ab73f86507d21e

Reply:
I would've liked it **not** to change this part. But you're proposition seems reasonable.

@sjoerdvisscher Do you feel like this is a deal-breaker?
I'm pretty neutral.

Reply:
No I prefer this downside over unformatted cabal files. It's especially something to keep in mind when we're going to format all cabal files at once.